wiki/Beatmap_Editor/Song_Setup/Primary_metadata_source/en.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - metadata
+  - song metadata
+  - source
+---
+
+# Primary metadata source
+
+An **primary metadata source** is a source for `Artist`, `Title`, and `Source` fields in a [beatmapset's metadata](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata) officially promoted by the artist or their label.
+
+An artist's website, their label's website, CD scans, or official uploads to websites like Bandcamp, YouTube, or SoundCloud are considered primary sources for song metadata. Third party websites such as wikis, databases (vndb, vgmdb, etc.), or music services (Spotify, iTunes, Amazon, etc.) are unreliable and therefore not used for song metadata.
```

Comment:
apparently needs a note about FA listing https://osu.ppy.sh/community/forums/topics/1051871

Reply:
shouldn't need this